templates/index.php

```diff
@@ -24,6 +24,7 @@
  *
  */
 style('mail', 'mail');
+style('mail', 'print');
```

Comment:
any chance we can inline these styles to the Vue components? We are trying to get the number of requests down in general

Reply:
ok, moved styles to `Message.vue`